.github/workflows/release.yml

```
@@ -0,0 +1,226 @@
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+        with:
+          registry: docker.elastic.co
+          secret: secret/apm-team/ci/docker-registry/prod
+          url: ${{ secrets.VAULT_ADDR }}
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+      - name: "Build docker image"
+        shell: bash
+        run: |
+          ./scripts/jenkins/build_docker.sh
+          ./scripts/jenkins/push_docker.sh
+
+  publish_aws_lambda:
+    name: "Publish AWS Lambda"
+    runs-on: ubuntu-latest
+    needs:
+      - await_artifact_on_maven_central
+    outputs:
+      arn_content: ${{ steps.arn_output.outputs.arn_content }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch }}
+      # TODO: actual lambda creation + upload
+      - name: Setup dummy ARN file
+        run: |
+          {
+              echo "### ARNs of the APM Java Agent's AWS Lambda Layer"
+              echo ''
+              echo '|Region|ARN|'
+              echo '|------|---|'
+          } > ".ci/arn-file.md"
+      - uses: actions/upload-artifact@v3
+        with:
+          name: arn-file
+          path: .ci/arn-file.md
+      - name: Add ARN file to output
+        id: arn_output
+        run: |
+          echo 'arn_content<<ARN_CONTENT_EOF' >> $GITHUB_OUTPUT
+          cat .ci/arn-file.md >> $GITHUB_OUTPUT
+          echo 'ARN_CONTENT_EOF' >> $GITHUB_OUTPUT
+  
+
+  create_github_release:
+    name: "Create GitHub Release"
+    needs:
+      - publish_aws_lambda
+      - update_major_branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch }}
+      - name: Await release-notes published
+        shell: bash
+        run: |
+          until .ci/release/wait_release_notes_published.sh ${{ inputs.version }}
+          do
+            echo "Release notes not published yet. Sleeping 30 seconds, retrying afterwards"
+            sleep 30s
+          done
+      - name: Compute major.x branch
+        id: get_dotx_branch
+        run: echo "dotx_branch=$(echo '${{ inputs.version }}' | sed -E 's/\..+/.x/')" >> $GITHUB_OUTPUT
+      - name: Create GitHub Release
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh release create v${{ inputs.version }} \
+            --title="Release ${{ inputs.version }}" \
+            --notes="[Release Notes for ${{ inputs.version }}](https://www.elastic.co/guide/en/apm/agent/java/current/release-notes-${{ steps.get_dotx_branch.outputs.dotx_branch }}.html#release-notes-${{ inputs.version }})
+            ${{ needs.publish_aws_lambda.outputs.arn_content }}"
+
+  bump_opbeans:
+    name: "Bump Opbeans agent version"
+    runs-on: ubuntu-latest
+    needs:
+      - await_artifact_on_maven_central
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: JonasKunz/opbeans-java
```

Oops, looks like a leftover from my side.
```suggestion
          repository: elastic/opbeans-java
```